--- ttbasic.c	Fri Feb 26 14:31:17 2016
+++ ttbasic_pico.c	Mon Feb 22 11:15:00 2021
@@ -6,4 +6,5 @@
 
-#include <stdlib.h>
-#include <time.h>
+#include <stdio.h>
+//#include <time.h>
+#include "pico/stdlib.h"
 
@@ -12,3 +13,6 @@
 int main(){
-	srand((unsigned int)time(0)); // for RND function
+	//srand((unsigned int)time(0)); // for RND function
+
+	stdio_init_all();
+
 	basic(); // call The BASIC
